wurst/objects/abilities/HostileAnimalAbilities.wurst

```diff
@@ -0,0 +1,79 @@
+package HostileAnimalAbilities
+
+// Standard lib Imports:
+import AbilityObjEditing
+import Assets
+import BuffObjEditing
+import ObjectIdGenerator
+import ObjectIds
+
+// Local Imports:
+import LocalObjectIDs
+import ToolTipsUtils
+
+let BEAR_BLOCK_CHANGE = 0.5
+let BEAR_BLOCK_DAMAGE_IGNORED = 7.
+// let BEAR_BLOCK_TT = "" +
+//     "This unit has hard skin; ith as {0} to reduce incoming damage by {1}"
+//     .format(
+//         BEAR_BLOCK_CHANGE.toToolTipOrange(),
+//         BEAR_BLOCK_DAMAGE_IGNORED.toToolTipOrange()
+//     )
+
+let WOLF_LIFE_STEAL_AMOUNT = 0.35
+// let WOLF_LIFE_STEAL_TT = "This unit restore {0} of his damage dealt as hit point"
+//     .format(WOLF_LIFE_STEAL_AMOUNT.toToolTipRed())
+
+
+// TODO: Display a buff indicating that the unit has a passive ability
```

Comment:
Why is the buff definition commented out if you're planning on using it?

Reply:
Hardened skin & wolf life steal abilities do not apply the buff, therefore it isn't displayed. It's commented because it didn't work as I wanted plus I didn't want to create more object definition, I would need to make a dummy ability, could actually just use `createDummyBuff()`,